scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1317,6 +1317,15 @@ Function SetEnvironmentVariable
     $env:Path="$env:Path;C:\Program Files\iotedge-moby;C:\Program Files\iotedge"
 }
 
+Function SetupMountedStorage
+{
+    If (!(Test-Path "C:\data\edgehub"))
```

Comment:
I will make var for this path when I make the same changes to master, then port it back.